pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1695,12 +1695,16 @@ self: super: {
   # Issue reported upstream, no bug tracker url yet.
   darcs = doJailbreak super.darcs;
 
-  # Too strict version bounds on base16-bytestring and http-link-header.
-  # This patch will be merged when next release comes.
-  github = appendPatch (pkgs.fetchpatch {
-    url = "https://github.com/phadej/github/commit/514b175851dd7c4a9722ff203dd6f652a15d33e8.patch";
-    sha256 = "0pmx54xd7ah85y9mfi5366wbnwrp918j0wbx8yw8hrdac92qi4gh";
-  }) super.github;
+  nix-thunk = appendPatches [
+    (pkgs.fetchpatch {
+      url = "https://github.com/obsidiansystems/nix-thunk/commit/49d27a85dd39cd9413c99958c67e596756a502b5.patch";
+      sha256 = "1p1n0123yrbdqyfk4kx3gq6bdv65l1bxgbsg51ckcwclg54xp2p5";
+    })
+    (pkgs.fetchpatch {
+      url = "https://github.com/obsidiansystems/nix-thunk/commit/512867c651977265d5d8f456b538f7a364ec8a8b.patch";
+      sha256 = "121yg26y4g28k8xv7y1j6c3pxm17vsjn3vi62kkc8g928c47yd02";
+    })
```

Comment:
@Ericson2314 I guess I'm a little late to this, but in the future could you make sure to include comments when you add patches like this?

If I'm a maintainer and I happen to be browsing through this file, I'll want to know the following things:

- What does the patch do?
- Why is it needed?
- When can it be deleted?
- Since you're upstream, is there some reason you can't just fix it upstream?

Reply:
Sorry. Those are just changing version bounds. Hopefully we make a Hackage revision soon, and then it goes away.

Reply:
Added in https://github.com/NixOS/nixpkgs/pull/152282

Reply:
Thanks!